src/types/sequence.rs

```diff
@@ -193,13 +189,16 @@ impl PySequence {
     where
         V: ToPyObject,
     {
-        let r =
-            unsafe { ffi::PySequence_Count(self.as_ptr(), value.to_object(self.py()).as_ptr()) };
-        if r == -1 {
-            Err(PyErr::fetch(self.py()))
-        } else {
-            Ok(r as usize)
+        fn inner(seq: &PySequence, value: PyObject) -> PyResult<usize> {
+            let r = unsafe { ffi::PySequence_Count(seq.as_ptr(), value.as_ptr()) };
```

Comment:
This isn't `error_on_minusone(..).map(|r| r as usize)`?

Reply:
`error_on_minusone` doesn't return a value, we could change it, but that's a future PR I think.